README.md

```diff
@@ -2,144 +2,56 @@
 
 A collection of composite GitHub Actions
 
-## Publish Build Scans® for pull-requests issued from forked repositories
+## Add a workflow summary to Maven builds 
 
 ### Description
-When submitting a pull request, a GitHub workflow that validates the change is usually triggered, however the Develocity Build Scans® can’t be published 
-as workflows from forked repositories do not have access to secrets although an access token is required to publish a Build Scan®
+The `maven-setup` action adds a workflow summary for each Maven invocation. 
+The summary can also be optionally added as a pull-request comment (if in a pull-request context). 
 
-This repository contains some actions which can be combined to solve this.
-
-_Note:_<br>
-The `Approve and Run` manual step documented [here](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks) must be enabled on the GitHub repository configuration to meet legal requirements (this is the default configuration).
+This action can be combined with the `maven-publish-build-scan` action to publish Build Scan® in a fork context. See [the relevant section](#publish-build-scans-for-pull-requests-issued-from-forked-repositories) for details.
 
-### Maven workflow
-
-#### Architecture
-![Architecture](./doc/architecture-maven.png)
-
-#### Usage
+### Usage
 
-In the GitHub workflow called to validate a pull-request, insert the `Setup Build Scan dump capture` step once in each job having steps invoking Maven.
+Insert the `Setup Maven` step once in each job having steps invoking Maven. Make sure to put the step before the Maven invocation.
 
 ```yaml
 name: PR Build
 jobs:
   build:  
-      - name: Setup Build Scan dump capture
+      - name: Setup Maven
```

Comment:
I know this is a bit late here, but I was just thinking, since this doesn't set up maven itself, perhaps the name is misleading. Does this add maven to the path?
I can't think of a better name, maybe `maven-summary-setup`? I wonder what @bigdaz would think here.

Reply:
We already discussed this in the [naming specification](https://docs.google.com/document/d/1CUt2IQIaBK6D7xihD2MT9_39Zvi3tRtuSW1dhnB7GuE/edit#heading=h.pr5426ll05no) and decided to keep it generic to allow further capabilities in the future (CI injection?)